src/Microsoft.ML.StandardTrainers/Standard/Online/AveragedPerceptron.cs

```diff
@@ -74,7 +74,7 @@ public sealed class AveragedPerceptronTrainer : AveragedLinearTrainer<BinaryPred
 
         /// <summary>
         /// Options for the <see cref="AveragedPerceptronTrainer"/> as used in
-        /// [AveragedPerceptron(Options)](xref:Microsoft.ML.StandardTrainersCatalog.AveragedPerceptron(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,Microsoft.ML.Trainers.AveragedPerceptronTrainer.Options).
```

Comment:
> [AveragedPerceptro [](start = 11, length = 19)

please Use xref instead of cref for method overloads: In addition to working in markdown, the second usage of xref is to create links with short anchortext. For methods with overloads (e.g. trainer extension methods with and without options), the cref text is verbose and human-unreadable. We use xref for those links everywhere, except in <seealso ..> where xref doesn't work.